src/components/EventTypeFilters.vue

```diff
@@ -1,12 +1,22 @@
 <template>
-  <div class="event-type-filters">
-    <h5 class="event-filters-title">Type of event</h5>
-    <div v-for="(label, type) in eventTypes">
-      <input type="checkbox"
-        class="no-margin-checkbox"
-        :checked="selectedEventTypes.includes(type)" 
-        @change="updateSelectedEventTypes(type, $event)"
-      /> {{label}}
+  <div>
+    <h5 v-if="showTitle" class="filter-events-title">Type of event</h5>
+    <div 
+      class="filter-events-item" 
+      v-for="(label, type) in eventTypes"
+    >
+      <div class="filter-events-checkbox">
```

Comment:
A good writeup here http://stackoverflow.com/a/27779075 --

> 1. You will need to hide the default checkbox control which is styled by your browser, and cannot be overridden in any meaningful way using CSS.

That answer is from 2015, I'm not sure if anything is different by now....probably I'm gonna avoid that rabbit hole for now but I will most def fix the accessibility issue. PR's to the situation later totally welcome if you have ideas :)